src/evaluate.ts

```diff
@@ -0,0 +1,60 @@
+import { BinaryOperationVertex, ReturnVertex, VertexKind, Value, ControlVertex, LiteralVertex, DataVertex } from 'graphir';
+import { GraphInterpreter } from './interpreter'
+import { BinaryOperation } from '../../TS-Graph-Extractor/src/types'
+
+export function evaluateControlNode(node: ControlVertex): Value {
```

Comment:
Technically control vertices shouldn't have values.
If this is meant for compound vertices, than they may be evaluated using evaluateDataNode because these are data vertices as well.